SPECS/orangefs/orangefs.spec

```diff
@@ -1,10 +1,20 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-Name: orangefs
-Version: 2.9.8
-Release: 1%{?dist}
-Summary: Parallel network file system client
-URL: https://www.orangefs.org/
+%global _hardened_build 1
+%global desc OrangeFS (formerly PVFS2) is a high-performance parallel \
+network file system designed for use on high performance computing \
+systems.  It provides very high performance access to disk storage for \
+parallel applications.  It is accessible through a variety of \
+interfaces, including the native OrangeFS library, the kernel, FUSE, \
+and MPI-IO. \
+\
+This package provides the pvfs2-client-core which is required to use \
+the kernel module.
+# Workaround for -fcommon issue
+# https://github.com/waltligon/orangefs/issues/80
+%define _legacy_common_support 1
```

Comment:
Seems unused in the spec. Let's try removing it and checking, if the package still builds.
```suggestion
```